src/Toolbar/Toolbar.react.js

```diff
@@ -292,6 +306,24 @@ class Toolbar extends PureComponent {
     focusSearchField() {
         this.searchFieldRef.focus();
     }
+    show = () => {
+        Animated.timing(this.state.moveAnimated, {
+            toValue: 0,
+            duration: 225,
+            easing: Easing.bezier(0.0, 0.0, 0.2, 1),
+            useNativeDriver: Platform.OS === 'android',
+        }).start();
+    }
+    hide = () => {
+        const { moveAnimated, styles } = this.state;
```

Comment:
Oh 😅, yeah that's still from the bottom navigation where you put the styles in the state.